docs/developer/local-docker.md

```diff
@@ -1,6 +1,17 @@
 # Running Narrative in local Docker
 
-It is useful to run the Narrative within a local docker container. E.g. this makes it easy to work on Narrative ui locally integrated with a local instance of kbase-ui.
+It is useful to run the Narrative within a local docker container. E.g. this makes it
+easy to work on Narrative ui locally integrated with a local instance of kbase-ui.
+
+## Requirements
+
+- make
+- git
+- docker
+- nodejs 16
+- nvm
+
+> Sorry, this is just a quick guide, no instructions provided for installing these.
```

Comment:
If people don't know how to install them, they shouldn't even be reading this file!

Reply:
That is true, but sometimes I do memorialize installation instructions, too, in the hope it imparts some experience to novices. I know many people still just go to, say, the nodejs site and download whatever they see. E.g. they are on macos and they don't use macports or homebrew (yet.)